### ISSUE TITLE
Siri in iOS 13 responds "that feature is not available..." when trying to turn on/off a wemo outlet

### ISSUE THREAD

**Issue by hepcat72** [rudders/homebridge-wemo#17]
After I installed iOS 13 recently, I noticed that I can no longer use Siri to turn on/off my Wemo outlets. She responds with phrases like:

"Hmm, that feature is not available for the \<outlet name\>"
"Hmm, the \<outlet name\> does not support that action"

I have tried specifying the outlets using a large variety of ways.  The only thing that works is if I specify "turn on/off the \<room name\>".  That is only useful when there's only 1 device in that room though.

I talked to Apple about this and they said to make sure my WeMo app is up to date (which it is).  The outlet firmware is up to date as well.  Perhaps there's some new device communication protocol?  I just checked and I've got `homebridge-wemo@0.0.2`, which appears to be current...

**Comment by IMNotMax** [comment]
Did you have wemo outlets with HomeKit enabled ? I had conflicts with homebridge registering wemo outlets and these very same outlets directly added in HomeKit. I found 2 solutions. First one, forget homebridge for these devices. Find the serial number (wemo app > more > settings and about > hardware info) and add it in the config.json as ignored devices. 
Second one, reset the device and let the “natural HomeKit initialisation” to disappear. Then setup the device with the wemo app and let the wemo platform in homebridge to discover the device(s).

**Comment by hepcat72** [comment]
I did not at first understand what you were talking about. I'd never messed with enabling HomeKit before, so I didn't know what "directly added in HomeKit" or "natural HomeKit initialisation" meant.

But you inspired me to search the settings app for "HomeKit" and it revealed a HomeKit screen in the Privacy settings that I'd never seen before (settings > privacy > homekit). There was a wemo toggle that was turned on (that I never turned on - so it must've been added either by a WeMo update, the iOS 13 update, or both). I toggled it off and all of a sudden, I could control my WeMo outlets!

There are also a few other entries, like the app for Liftmaster, though that doesn't seem to disrupt my homebridge garage door controls, even though it's turned on.

Thanks for the help.  I will close this issue.
